src/battle-tooltips.ts

```diff
@@ -476,14 +479,9 @@ class BattleTooltips {
 				});
 				zEffect = this.getStatusZMoveEffect(move);
 			} else {
-				let moveid;
-				if (item.zMoveType) {
-					moveid = BattleTooltips.zMoveTable[item.zMoveType as TypeName];
-				} else {
-					moveid = BattleTooltips.maxMoveTable[move.type as TypeName];
-				}
+				let moveid = BattleTooltips.zMoveTable[item.zMoveType as TypeName];
```

Comment:
nitpick but this should be called `moveName` since the moves in the zMoveTable aren't IDs